deltachat-ffi/deltachat.h

```diff
@@ -2095,6 +2095,10 @@ int             dc_may_be_valid_addr         (const char* addr);
  * To validate an e-mail address independently of the contact database
  * use dc_may_be_valid_addr().
  *
+ * If there are multiple contacts with this address
```

Comment:
I think the whole documentation should be improved, currently the first line says: "Check if an e-mail address belongs to a known and unblocked contact.". It doesn't just check, but looks up a contact.